code/modules/research/rdmachines.dm

```diff
@@ -129,3 +131,6 @@
 	overlays += "[initial(name)]_[stack_name]"
 	sleep(10)
 	overlays -= "[initial(name)]_[stack_name]"
+
+/obj/machinery/r_n_d/proc/check_mat(datum/design/being_built, var/M)
+	return 0
```

Comment:
```suggestion
	return FALSE
```

Reply:
this is the number of copies of the design you can build, should I still change it to FALSE or add a comment?

Reply:
Comment please